MANIFEST.MF

```diff
@@ -1 +1 @@
-Main-Class: Scatt
+Main-Class: ScattReport
```

Comment:
The manifest is for the jar creation - the main class should stay as Scatt, since that's where our main method will be.